index.js

```diff
@@ -622,6 +622,28 @@ function base64Slice (buf, start, end) {
   }
 }
 
+function decodeLargeCodePointsArray (array) {
+  var res = ''
+  var i
```

Comment:
I was following the style of the rest of the code; I thought y'all were var-hoisters. :)